package.json

```diff
@@ -53,15 +54,13 @@
     "imagemin-pngquant": "^6.0.1",
     "is-valid-glob": "^1.0.0",
     "journalize": "^2.5.1",
-    "request": "^2.88.2",
+    "prettier": "^2.8.8",
     "sass": "^1.49.8"
   },
   "dependencies": {
     "@popperjs/core": "^2.11.2",
-    "ansi-colors": "^4.1.3",
```

Comment:
Moved these packages to `dev-dependencies`.

Reply:
That definitely needed to be done.